python/ray/serve/tests/test_cli.py

```diff
@@ -528,10 +529,10 @@ def test_idempotence_after_controller_death(ray_start_stop, use_command: bool):
     deploy_response = subprocess.check_output(["serve", "deploy", config_file_name])
     assert success_message_fragment in deploy_response
 
-    ray.init(address="auto", namespace=SERVE_NAMESPACE)
+    ray_context = ray.init(address="auto", namespace=SERVE_NAMESPACE)
     serve.start(detached=True)
     wait_for_condition(
-        lambda: len(ray.util.list_named_actors(all_namespaces=True)) == 4, timeout=15
+        lambda: len(list_actors(address=ray_context["address"])) == 4, timeout=15
```

Comment:
Nit: this is deprecated
```
DeprecationWarning: Accessing values through ctx["address"] is deprecated. Use ctx.address_info["address"] instead.
```

Reply:
Thanks for the catch! I updated the deprecated accesses.